src/tests/settings_tests.ts

```diff
@@ -52,10 +52,11 @@ test("helper function checkValueAndDescriptionOnObject should assert when given
 	ok(checkValueAndDescriptionOnObject(valid));
 });
 
-test("All entries in setting.json should have a non-empty Value and Description", () => {
+// TODO is a non-empty check necessary?
+/*test("All entries in setting.json should have a non-empty Value and Description", () => {
```

Comment:
Yeah, wasn't sure if there was a technical reason why we disallow empty values... because now I'd like to start using empty values. I'll assume it's all good and delete this test.

#Pending